### PR TITLE
Add Peaka Destination Documentation

### PR DESCRIPTION
**Pull request by kobalski**
### Proposed changes
Documentation for Direct Destination integration for Peaka.

### Merge timing
- ASAP once approved.

### CODE REVIEW

File: src/connections/destinations/catalog/peaka/index.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+---
+title: Peaka Destination
+---
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/peaka/index.md:3]:
```suggestion
---
title: Peaka Destination
id: 651ea97b7982672f1d66b93c
beta: true
---
```

File: src/connections/destinations/catalog/peaka/index.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+[Peaka](https://peaka.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="\_blank”} is a Zero-ETL platform that connects to any source. Ingest high-volume event and streaming data. Replace batch with real-time access.
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/peaka/index.md:5]:
```suggestion
[Peaka](https://peaka.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} is a Zero-ETL platform that connects to any source. Ingest high-volume event and streaming data. Replace batch with real-time access.
```

File: src/connections/destinations/catalog/peaka/index.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+By integrating Peaka with their Segment workspace, users can designate Peaka as one of their destinations. This means that events such as pages, screens, tracks,
+and more will be sent directly to Peaka's Segment data catalog. With this integration, Peaka users can begin querying their product events.
+
+This destination is maintained by Peaka. For any issues with the destination, [contact the Peaka Support team](mailto:info@peaka.com).
+
+## Getting started
+
+1. From your workspace's [Destination catalog page](https://app.segment.com/goto-my-workspace/destinations/catalog){:target="\_blank”} search for **Peaka**
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/peaka/index.md:14]:
```suggestion
1. From your workspace's [Destination catalog page](https://app.segment.com/goto-my-workspace/destinations/catalog){:target="_blank”} search for **Peaka**
```

File: src/connections/destinations/catalog/peaka/index.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+2. Select **Peaka** and click **Add Destination**
+3. Choose which Source should send data to the **Peaka** destination.
+4. Go to the [Peaka](https://peaka.studio/){:target="\_blank"}, create your Segment Catalog and copy the **API key**.
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/peaka/index.md:17]:
```suggestion
4. Go to the [Peaka](https://peaka.studio/){:target="_blank"}, create your Segment Catalog and copy the **API key**.
```

File: src/connections/destinations/catalog/peaka/index.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+- [Getting started](#getting-started)
+- [Supported methods](#supported-methods)
+  - [Page](#page)
+  - [Screen](#screen)
+  - [Identify](#identify)
+  - [Track](#track)
+  - [Group](#group)
+  - [Alias](#alias)
+
+<!-- /code_chunk_output -->
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/peaka/index.md:113]: Not sure where you got this? But I would remove unless you had something specific in mind

**Review by tcgilbert — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/peaka/index.md (hunk 5 above)
Comment (kobalski) [reply src/connections/destinations/catalog/peaka/index.md]: @tcgilbert This table of contents section is auto generated by a VSCode markdown extension that I use. It is not important so I deleted it. Could you check again?

**Review by tcgilbert — APPROVED** [review]

**Review by stayseesong — APPROVED** [review]

**Pull request merged**